util/encodings.go

```diff
@@ -90,3 +91,13 @@ func AllAlternateNameWithTagAreIA5(ext *pkix.Extension, tag int) (bool, error) {
 
 	return true, nil
 }
+
+// IsEmptyASN1Sequence returns true if
+// *input is an empty sequence (0x30, 0x00) or
+// *len(inout) < 2
+// This check covers more cases than just empty sequence checks but it makes sense from the usage perspective
+var emptyASN1Sequence = []byte{0x30, 0x00}
```

Comment:
This can probably be marked `const`.

Reply:
slices/maps/pointers can't be marked as constant.

Reply:
ah hah! Thanks.